specification/netapp/NetApp.Management/Snapshot.tsp

```diff
@@ -0,0 +1,103 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./Volume.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.NetApp;
+/**
+ * Snapshot of a Volume
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/empty-updateable-properties" "For backward compatibility"
+@parentResource(Volume)
+model Snapshot is Azure.ResourceManager.ProxyResource<SnapshotProperties> {
+  ...ResourceNameParameter<
+    Resource = Snapshot,
+    KeyName = "snapshotName",
+    SegmentName = "snapshots",
+    NamePattern = ""
+  >;
+
+  /**
+   * Resource location
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+  @visibility(Lifecycle.Read, Lifecycle.Create)
+  location: string;
+}
+
+/**
+ * Snapshot of a Volume
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "For backward compatibility"
+model SnapshotPatch {}
```

Comment:
In Swagger, the payload to PATCH is written as
```
    "snapshotPatch": {
      "description": "Snapshot patch",
      "type": "object",
      "properties": {}
    },
```

The intention appears to be an empty model.

Also, in the example JSON, the payload is `{}`.

Reply:
example JSON uses empty JSON
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/netapp/resource-manager/Microsoft.NetApp/stable/2025-06-01/examples/Snapshots_Update.json#L10